src/universal/auth/pages/ChangePasswordPage/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {reduxForm} from 'redux-form';
+import {ErrorMessage, FormattedMessage, FormTextField, RaisedButton} from 'universal/common/components';
+const {required, repeatPasswordValidation} = require('er-common-components/lib/helpers/validation');
+
+export const CHANGE_PASSWORD_FORM_NAME = 'CHANGE_PASSWORD_FORM_NAME';
+
+@reduxForm({
+  form: CHANGE_PASSWORD_FORM_NAME,
+})
+export default class ChangePasswordForm extends React.Component<any, any> {
+  public static propTypes = {
+    handleSubmit: React.PropTypes.func.isRequired,
+    isLoading: React.PropTypes.bool,
+    errorMessage: React.PropTypes.string,
+  };
+
+  public render() {
+    const {
+      isLoading,
+      errorMessage,
+      handleSubmit,
+    } = this.props;
+
+    // TODO: Add validation that makes sure both password fields have the same value (repeatPasswordValidation?)
+    // TODO: Add password rules
+    return (
+      <form onSubmit={handleSubmit}>
+        <div>
+          <FormTextField
+            withRef
+            ref="password"
+            fullWidth
+            disabled={isLoading}
+            name="password"
+            type="password"
+            hintText={<FormattedMessage id="App.password" />}
+            floatingLabelText={<FormattedMessage id="App.password" />}
+            validate={[required, repeatPasswordValidation]}
```

Comment:
https://github.com/liviuignat/heimdall/blob/develop/src/universal/auth/pages/RegisterPage/RegisterForm.tsx#L64
```ts
repeatPasswordValidation(passwordFieldName: string): (value: any, formData: AllFormFields) => string | null
```

The repeatPasswordValidation returns a callback and it will look for the value to compare in `formData[passwordFieldName]`.